packages/gradbench/src/App.tsx

```diff
@@ -3,38 +3,38 @@ import "./App.css";
 const App = () => {
   return (
     <>
-      <div className="header-cont">
-        <a href="https://github.com/gradbench/gradbench" target="_blank">
-          <h1 className="header">
-            GradBench
-            <span className="subtitle">
-              A Benchmark for Differentiable Programming Across Languages and
-              Domains
-            </span>
-            {/* <a className="repo" href="https://github.com/gradbench/gradbench">
-            GitHub Repository
-          </a> */}
-          </h1>
-        </a>
-      </div>
+      <a href="https://github.com/gradbench/gradbench" target="_blank">
+        <h1 className="header">
+          GradBench
+          <span className="subtitle">
+            A Benchmark for Differentiable Programming Across Languages and
+            Domains
+          </span>
+        </h1>
+      </a>
 
-      {/* <h2>Automatic Differentiation</h2>
-      <p>
-        Automatic Differentiation (AD) is the process of differentiation a
-        mathematical function with respect to its inputs. Fields such as
-        interactivity and machine learning benefit greatly when programs can
-        automatically compute derivatives.{" "}
-      </p> */}
+      {/* Appears for mobile view only */}
+      <p className="hidden">
+        <b style={{ color: "#3e4756" }}>
+          A Benchmark for Differentiable Programming Across Languages and
+          Domains
+        </b>
+      </p>
 
       <div className="row">
         <h2 className="subheading">Benchmarking</h2>
         <div className="fillspace"></div>
       </div>
       <p className="text">
-        GradBench is designed to work across languages and domains. Functions
-        are written in <b style={{ color: "#3e4756" }}>Adroit</b> and
-        automatically translated to a tool's native language. The architecture
-        allows for the easy addition of new functions and tools.
+        <li> GradBench is designed to work across languages and domains. </li>
+        <li>
+          Functions are written in <b style={{ color: "#3e4756" }}>Adroit</b>{" "}
+          and automatically translated to a tool's native language.
+        </li>
+        <li>
+          The architecture allows for the easy addition of new functions and
+          tools.
+        </li>
```

Comment:
Having these as bullets is OK, but putting the `<li>` nodes inside of a `<p>` instead of a `<ul>` doesn't work; the indentation is broken when the `<li>` is split across multiple lines, like on mobile. I'm guessing that the reason you don't use `<ul>` is because this part of your CSS causes the bullets to disappear:
```css
* {
  margin: 0;
}
```
Can you modify your CSS so that it works? I'm guessing that you'll just need to replace the `*` with something more specific, or you could apply your own styling to `<ul>` or `<li>` to make it work.